test/elixir_internal_certificate_web/helpers/icon_helper_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule ElixirInternalCertificateWeb.IconHelperTest do
   alias ElixirInternalCertificateWeb.IconHelper
 
   describe "icon_tag/3" do
-    test "renders a svg icon" do
+    test "when svg icon is available, it should render" do
```

Comment:
do we need to rename this 🤔 with the `when SVG icon is available` it means we also have the case `when SVG icon is unavailable` will you plan to add that @nvminhtue :D